BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTests.cs

```diff
@@ -281,7 +282,7 @@ public async void TieUserWithLandlord_WithNonExistentLink_RedirectToInvite()
         result.Should().BeOfType<RedirectToActionResult>().Which.ActionName.Should().Be("Invite");
         result.Should().BeOfType<RedirectToActionResult>().Which.RouteValues.Should()
             .Contain("inviteLink", inviteLink);
-        UnderTest.TempData["FlashMessage"].Should().BeNull();
+        UnderTest.TempData["FlashMessages"].Should().BeNull();

```

Comment:
Should this just use the FlashMessages?

Reply:
That would be another one I missed because the test was passing (I'm now searching to check that all singular instances of "FlashMessage" are dealt with!)

Reply:
Having just checked, that seems to have already been changed so all is fine.